Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java

```diff
@@ -57,25 +59,28 @@
     private Map<Template, List<Process>> templatesToCreate = new HashMap<>();
     private Map<Template, Template> matchingTemplates = new HashMap<>();
     private MigrationService migrationService = ServiceManager.getMigrationService();
+    private boolean migrateWorkflow = true;
 
     /**
      * Migrates the meta.xml for all processes in the database (if it's in the
      * old format).
      *
      */
     public void migrateMetadata() {
+        migrateWorkflow = false;
         try {
-            migrationService.migrateMetadata();
+            allProjects = ServiceManager.getProjectService().getAll();
+            projectListShown = true;
         } catch (DAOException e) {
-            Helper.setErrorMessage(ERROR_DATABASE_READING, new Object[] {ObjectType.PROCESS.getTranslationSingular() },
-                logger, e);
+            Helper.setErrorMessage("Error during database access");
```

Comment:
Why did you remove the `logger` and `e` parameters from method call? Maybe / helpful information get lost.

Reply:
I deleted the whole method and put another method here, which has a catch in the same place, but another helper call. This only looks like the diff view, as if I had just changed the one line. The modification process was different, so I did not notice.